### PR TITLE
fix list refresh

### PR DESCRIPTION
**Pull request by minh-rakuten**
# Description
Pull-down refresh for mini app list.
Reloading the list when go back.

## Links
[MINI-818](https://jira.rakuten-it.com/jira/browse/MINI-818)

# Checklist
- [ ] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [ ] I ran `./gradlew check` without errors

### CODE REVIEW

File: testapp/build.gradle [hunk 1]
```diff
@@ -54,6 +54,7 @@ dependencies {
     implementation "com.github.bumptech.glide:glide:$glide"
     annotationProcessor "com.github.bumptech.glide:compiler:$glide"
     implementation "com.google.android.material:material:$material"
+    implementation "androidx.swiperefreshlayout:swiperefreshlayout:1.0.0"
```
Comment (ts-nitish-a-jain) [inline testapp/build.gradle:57]: Let's use the same way how other dependencies are added. What do you think?

File: testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/miniapplist/MiniAppListFragment.kt [hunk 2]
```diff
@@ -48,12 +49,21 @@ class MiniAppListFragment : BaseFragment(), MiniAppList {
         return binding.root
     }
 
+    override fun onStart() {
+        super.onStart()
+        srl.post {
```
Comment (ts-nitish-a-jain) [inline testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/miniapplist/MiniAppListFragment.kt:54]: I think we can something more descriptive please. `srl` on google gives me "SRL Diagnostics is a diagnostic company" 🎉

**Comment by minh-rakuten** [comment]
Adding showing version tag. [MINI-111](https://jira.rakuten-it.com/jira/browse/MINI-111) [MINI-557](https://jira.rakuten-it.com/jira/browse/MINI-557)

File: testapp/src/main/res/layout/item_list_miniapp.xml [hunk 3]
```diff
@@ -102,9 +101,8 @@
                 android:alpha="0.6"
                 android:textColor="@android:color/black"
                 android:textSize="@dimen/text_medium_14"
-                android:text="@string/lb_version_id"
+                android:text="@{miniapp.version.versionTag}"
```
Comment (ts-nitish-a-jain) [inline testapp/src/main/res/layout/item_list_miniapp.xml:104]: I'm not sure about this. Do we still need to display it?

Comment (minh-rakuten) [reply testapp/src/main/res/layout/item_list_miniapp.xml:104]: Yeah we still need it. It's from spec [MINI-111](https://jira.rakuten-it.com/jira/browse/MINI-111) [MINI-557](https://jira.rakuten-it.com/jira/browse/MINI-557)

Comment (ts-nitish-a-jain) [reply testapp/src/main/res/layout/item_list_miniapp.xml:104]: I hope so. @corycaywood Could you double confirm this pls?

Comment (corycaywood) [reply testapp/src/main/res/layout/item_list_miniapp.xml:104]: I think that's right. The version tag should be user visible.

**Pull request merged**